WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1257,7 +1257,7 @@ extension WooAnalyticsEvent {
 
         enum CardReaderType: String {
             case external
-            case builtIn = "built_in"
+            case tapToPay = "built_in"
```

Comment:
I'm guessing this is deliberately left, to keep the analytics comparable over time?

Reply:
Yes. I kept all the analytics strings deliberately unchanged.